modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Checkbox/Checkbox.tsx

```diff
@@ -140,7 +140,7 @@ const Main = ({
 				checked={
 					!!(
 						value ??
-						(predefinedValue.length
+						(typeof predefinedValue === 'object'
```

Comment:
For a matter of clarity, you better use `Array.isArray()` to check if a variable is an array
> You can securely check if a given object is, in fact, an Array using `Array.isArray(myObj)`
[ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof)

[documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray)

Sorry,  I didn't point out that before because I just realized it.